docs/download_config.md

```diff
@@ -1,51 +1,42 @@
 # Download kops config spec file
+
 KOPS operates off of a config spec file that is generated during the create phase.  It is uploaded to the amazon s3 bucket that is passed in during create.
 
 If you download the config spec file on a running cluster that is configured the way you like it, you can just pass that config spec file in to the create command and have kops create the cluster for you , `kops create -f spec_file` in a completely unattended manor.
 
-## How to download the config spec file. ( 2 possible ways )
-
-Let's say you create your cluster with the following configuration options:
+Let us say you create your cluster with the following configuration options:
 
 ```
 declare -x KOPS_STATE_STORE=s3://k8s-us-west
 declare -x CLOUD=aws
 declare -x ZONE="us-west-1a"
 declare -x MASTER_ZONES="us-west-1a"
-declare -x NAME=westtest.c.foo.com
-declare -x K8S_VERSION=1.4.6
+declare -x NAME=k8s.example.com
+declare -x K8S_VERSION=1.6.4
```

Comment:
`declare -x` is equivalent to `export`, except it's more obscure and not available in `sh`, perhaps swap them out for `export`s?